src/index.html

```diff
@@ -8,9 +8,44 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form big_form"
```

Comment:
Pay attention to the class names, fix it according to the checklist <img width="1044" alt="Screenshot 2022-11-09 at 14 44 23" src="https://user-images.githubusercontent.com/104769168/200833616-06b42a89-5e27-4e32-8bae-4fbd1609bb5a.png">